Terminal.Gui/Core/TextFormatter.cs

```diff
@@ -154,9 +154,15 @@ public class TextFormatter {
 		/// </summary>
 		public bool AutoSize { get; set; }
 
+		/// <summary>
```

Comment:
```
Gets or sets a flag that determines whether <see cref="Text"/> will have trailing spaces preserved or not when <see cref="WordWrap"/> is enbled. If `true` any trailing spaces will be trimmed when either the <see cref="Text"/> property is changed or when <see cref="WordWrap"/> is set to `true`. The default is `false`.
```
This makes it clear to the user a) that this controls something that changes the contents of the control, and b) when it does so.

Reply:
I agree, thanks. It will looks like this:

```cs
		/// <summary>
		/// Gets or sets a flag that determines whether <see cref="Text"/> will have trailing spaces preserved
		/// or not when <see cref="WordWrap"/> is enabled. If `true` any trailing spaces will be trimmed when
		/// either the <see cref="Text"/> property is changed or when <see cref="WordWrap"/> is set to `true`.
		/// The default is `false`.
		/// </summary>
```